book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Let's add also a method that allows us to add the many-to-many relationship from the Book and the Person class. This will facilitate sometimes creating rentals. Since we need to initialize an instance of the Rental class anyway, we can use it to help ourselves.

The method should accept then, a person, a date and instantiate a Rental.

```bash
def add_rental(person, date)
  Rental.new(person, date, self)
end
```

And this will do the trick! :sweat_smile: